File: java/pl/edu/pg/mygame/organisms/plants/SowThistle.java
package pl.edu.pg.mygame.organisms.plants;

import pl.edu.pg.mygame.Grid;
import pl.edu.pg.mygame.organisms.Species;

import java.awt.*;

public class SowThistle extends Plant {

    public SowThistle(Point p) {
        super(p);
        this.species = Species.SOW_THISTLE;
    }

    @Override
    public void makeAction(Grid grid) {
        for (int i = 0; i < 3; ++i)
            super.makeAction(grid);
    }
}

File: python/organisms/plants/sow_thistle.py
from organisms.plants.plant import Plant
from organisms.species import Species


class SowThistle(Plant):
    def __init__(self, pos):
        super().__init__(pos)
        self.species = Species.SOW_THISTLE

    def make_action(self, grid):
        for _ in range(3):
            super().make_action(grid)
